features/install-cli.feature

```diff
@@ -48,6 +48,20 @@ Feature: Command behavior of client side tools installation
     When I run `bundle exec vagrant service-manager install-cli docker --cli-version 111.222.333`
     Then the exit status should be 126
 
+    When I run `bundle exec vagrant service-manager install-cli docker`
+    And box is "adb"
```

Comment:
Actually I think it will work, but by chance, since these are the last tests in this file.